Tests/Factory/DiactorosFactoryTest.php

```diff
@@ -161,4 +161,33 @@ public function testCreateResponseFromBinaryFile()
 
         $this->assertEquals('Binary', $psrResponse->getBody()->__toString());
     }
+
+    public function testUploadErrNoFile()
+    {
+        $file = new UploadedFile('', '', null, 0, UPLOAD_ERR_NO_FILE, true);
+        $this->assertEquals(0, $file->getSize());
+        $this->assertEquals(UPLOAD_ERR_NO_FILE, $file->getError());
+        $this->assertFalse($file->getSize());
```

Comment:
Don't you mean a second argument to `assertFalse`? It would be a third argument for `assertEquals`, but the comment you linked to erroneously mentioned `assertFalse`.

Assert false [doesn't take a third argument](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertFalse).